src/main/java/seedu/address/logic/LogicManager.java

```diff
@@ -62,7 +62,7 @@ public ReadOnlyAddressBook getAddressBook() {
 
     @Override
     public ObservableList<Person> getFilteredPersonList() {
```

Comment:
Return type can be ObservableList\<Patient>
Method name getFilteredPatientList()